src/main/java/ch/njol/skript/expressions/ExprEnchantments.java

```diff
@@ -109,36 +124,54 @@ public void change(Event e, @Nullable Object[] delta, ChangeMode mode) {
 		
 		switch (mode) {
 			case ADD:
-				for (ItemType item : source)
-					item.addEnchantments(enchants);
+				for (ItemType item : source) {
+					if (isStored)
+						item.addStoredEnchantments(enchants);
+					else
+						item.addEnchantments(enchants);
+				}
```

Comment:
```suggestion
				if (isStored) {
					for (ItemType item : source)
						item.addStoredEnchantments(enchants);
				} else {
					for (ItemType item : source)
						item.addEnchantments(enchants);
				}
```
Longer code but we can avoid checking the if statement repeatedly